syft/execution/role.py

```diff
@@ -34,6 +35,7 @@ def __init__(
         output_placeholder_ids: Tuple[int, str] = None,
         # General kwargs
         id: Union[str, int] = None,
+        operations: Dict[str, List[Action]] = None
```

Comment:
Operations here is a `dict` containing the `list` of the corresponding `Action`s translated into other frameworks. I believe this requires us to update syft-proto